opensearch_reports/service.py

```diff
@@ -38,25 +39,21 @@ def is_sync_disabled(self):
             # If no dashboard entry, assume sync is enabled
             return False
 
-    def update(self, thing, action, *args, refresh=None, using=None, **kwargs):
```

Comment:
I would recommend keeping overriding 'update' method due to 'Enable/Disable OpenSearch synch' which is one of the functional requirement and features in the system (we can enable/disable opensearch dashboard). Please revert whole update method. By deleting 'update' we remove this functionality (update works for singular CRUD operation like adding new program/edit new program, edit individual data etc).

Reply:
`update` is called from `bulk` in django_elasticsearch_dsl https://github.com/django-es/django-elasticsearch-dsl/blob/master/django_elasticsearch_dsl/documents.py#L245-L263 so checking is_sync_disabled in `bulk` alone should suffice.